### ISSUE TITLE
RStudio code completion is slow to get started

### ISSUE THREAD

**Issue by lucazav** [Azure/azureml-sdk-for-r#259]
After refreshing the azuremlsdk package in RStudio on my Compute Instance, I loaded the library, but I can't see any suggestion from code completion:

![image](https://user-images.githubusercontent.com/6104437/75430887-4815db80-594c-11ea-8294-79fe3be16873.png)

Are there file system performance issues?

-----------------------------------------------------------
**UPDATE**

The code completion works fine:

![image](https://user-images.githubusercontent.com/6104437/75432099-2d446680-594e-11ea-9198-03fcf837b1ab.png)

But I still can't see _azureml_ module functions so I can't navigate them.
Is there a web reference to navigate instead?

**Comment by diondrapeck** [comment]
Hi @lucazav - I can't reproduce. 

Can you try (if you haven't already) once the azureml autocomplete box comes up, press tab to complete the suggestion, then a "$"? When I do that, I get the expected azureml model functions listing.

**Comment by lucazav** [comment]
Hi @diondrapeck, I already tried, but nothing.
Tha strage thing is that it seems something went wrong when I tried to re-install the _azuremlsdk_ package and to update the underlying Python lib. In fact if I run the command `azureml$core$authentication$InteractiveLoginAuthentication(tenant_id = '<my-tenant-id>')`, it gives me an error.
I did the same steps on my RStudio on-premises and all works fine.
Do I to reinstall the library again in the Compute Instance?

Thanks
